t/api/15-users.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+# Copyright (C) 2017-2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use Date::Format;
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+use OpenQA::Test::Case;
+use OpenQA::Client;
+require OpenQA::Schema::Result::Jobs;
+
+OpenQA::Test::Case->new->init_data(fixtures_glob => '03-users.pl');
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+my $app = $t->app;
+$t->ua(
+    OpenQA::Client->new(apikey => 'ARTHURKEY01', apisecret => 'EXCALIBUR')->ioloop(Mojo::IOLoop->singleton));
+$t->app($app);
+
+subtest 'Admins can delete users' => sub {
+    $t->delete_ok("/api/v1/user/99904")->status_is(200, "delete user successfully.");
```

Comment:
I see no need for the subtest. You can just extend the test description string in this line, e.g. `'admins can delete users'`. And please single ticks (as you don't want to evaluate variables in the string) and no dot at EOL